compiler/src/main/kotlin/software/amazon/lastmile/kotlin/inject/anvil/processor/ContributesBindingProcessor.kt

```diff
@@ -56,17 +60,29 @@ import kotlin.reflect.KClass
 internal class ContributesBindingProcessor(
     private val codeGenerator: CodeGenerator,
     override val logger: KSPLogger,
+    options: Map<String, String>,
 ) : SymbolProcessor, ContextAware {
 
+    private val ignoreUnsupportedParameters: Boolean by lazy {
+        options[OPTION_IGNORE_ANVIL_UNSUPPORTED_PARAM_WARNINGS] == "true"
+    }
+
     private val anyFqName = Any::class.requireQualifiedName()
 
     override fun process(resolver: Resolver): List<KSAnnotated> {
         resolver
-            .getSymbolsWithAnnotation(ContributesBinding::class)
+            .getSymbolsWithAnnotations(
+                ContributesBinding::class,
+                com.squareup.anvil.annotations.ContributesBinding::class,
+                com.squareup.anvil.annotations.ContributesMultibinding::class,
```

Comment:
This should be an opt-in and not happen automatically. Our codebase was in a temporary state where we used kotlin-inject and Dagger 2 simultaneously. Not having an opt-out here would have caused problems for us.

Reply:
Moving to a separate module would make it inherently opt-in, so that'd be fixed by the proposed new route.